server/http.go

```diff
@@ -132,36 +135,32 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	tok, err := conf.Exchange(ctx, code)
+	token, err := conf.Exchange(context.Background(), code)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	zoomUser, err := p.getZoomUserWithToken(tok)
-
+	zoomUser, err := zoom.GetUserViaOAuth(token, conf, p.getZoomAPIURL())
```

Comment:
We don't have access to the `OAuthClient` at this point, so we cannot call `client.GetUser` here. That's why I had to `GetUserViaOAuth` and use it directly.

Reply:
@larkox That makes a lot of sense. Done in https://github.com/mattermost/mattermost-plugin-zoom/pull/150/commits/ca90a1b928cb6f5fa163f4488dc646ec4ab8afda